### ISSUE TITLE
[BUG] results are not matching as published on the arcgis experience demo page 

### ISSUE THREAD

**Issue by sumanttyagi** [LADI-Dataset/ladi-overview#17]
i tried few examples there is huge disconnect in results :
i just download the image and ran small as mentioned in read me model  using below code 
![image](https://github.com/user-attachments/assets/f89c8690-4545-4e84-b82a-7daba35c2f2a)

as well as small - reference just to try it 
![image](https://github.com/user-attachments/assets/bfde951e-0bd9-437a-b806-cfa318bf1d71)

on portal -https://experience.arcgis.com/experience/12facb85c45a4b7e9c4caef04be6e4d2/?draft=true#data_s=id%3AdataSource_2-18f59fc7c23-layer-2%3A111
it shows debris any as 0.67 which is incorrect with actual output 
![image](https://github.com/user-attachments/assets/fa9342ce-5e9f-49f1-92bc-776aab74b51a)
![image](https://github.com/user-attachments/assets/918db21a-bf62-4d9e-87d8-71aa7839f296)

Am I missing something? Are these results incorrect? For small variants, the outputs do not match. Is any preprocessing required?

**Comment by jeffliu-LL** [comment]
You are correct - I am unable to reproduce the weights published in the arcgis experience for that image. It is possible we ran the inference on those images using a different checkpoint. We can look into it and update the arcgis visualization to use the latest published versions of the models.

**Comment by sscheele** [comment]
Yeah, this is likely what happened. The AGOL experience data was added over the course of a few months and we updated the model at least once that I can remember during the same period, so probably this output was produced by a different set of weights. It looks to me like the OP's code is correct. Until we can rerun the AGOL inputs (or remove points added before the most recent model update), we should probably update the README to reflect the fact that some outputs in AGOL might not be the same as those from the most recent model to avoid confusion. Thanks for the bug!
